libfec/src/fec.c

```diff
@@ -0,0 +1,65 @@
+/* Utility routines for FEC support
+ * Copyright 2004, Phil Karn, KA9Q
+ */
+
+#include "fec.h"
+
+unsigned char Partab[256];
```

Comment:
Don't use this.  CPU is much cheaper than RAM for us.  Use [this](https://graphics.stanford.edu/~seander/bithacks.html#ParityParallel) implementation or the previous one to replace fec.h:parity().